<launch>
  <include file="$(find turtlebot_bringup)/kinect.launch" />

  <node pkg="turtlebot_calibration" type="scan_to_angle.py" name="scan_to_angle" >
    <param name="min_angle" value="-0.3" />
    <param name="max_angle" value="0.3" />
  </node>

  <node pkg="turtlebot_calibration" type="calibrate.py" name="turtlebot_calibration" output="screen" required="true">
    <remap from="cmd_vel" to="turtlebot_node/cmd_vel" />
    <remap from="imu" to="imu/raw" />
    <remap from="scan_angle" to="scan_angle" />
    <remap from="odom" to="odom" />
  </node>

</launch>